src/protocol.ts

```diff
@@ -55,6 +55,11 @@ export interface ActionableMessage {
 	commands?: Command[];
 }
 
+export interface ClasspathParams {
```

Comment:
This interface name is not clear from context. Should be ClasspathResolveRequestParams